### PR TITLE
GitHub workflow to merge this branch into 2022.2

### PR DESCRIPTION
**Pull request by sergej-koscejev**
Adapted from https://github.com/modelix/modelix.mps/blob/mps/2022.2/.github/workflows/PR-into-2022-2.yml

Should create a merge PR for the 2022.2 branch whenever a PR is merged into this branch.

### CODE REVIEW

File: .github/workflows/PR-into-2022-2.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+          
+          # actual merge operation
+          git merge --no-ff --message "Merge 2021.3 -> 2022.2" origin/maintenance/mps20213 -- || ( echo "Merge failed. Please merge manually!" ; exit 1 )
+
+      # actually merge from the previous branch
+      - name: Prepare environment variables for PR
+        run: |
+          # strip the title of previous automatic PR content
+          NEW_TITLE='${{ github.event.pull_request.title }}'
+          NEW_TITLE="${NEW_TITLE// (2021.2 -> 2021.3)}"
```
Comment (alexanderpann) [inline .github/workflows/PR-into-2022-2.yml:40]: The title looks wrong.

File: .github/workflows/PR-into-2022-2.yml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+          echo "NEW_TITLE=$NEW_TITLE" >> $GITHUB_ENV
+
+      # create a PR based on the merge
+      - name: Create PR for MPS 2022.2
+        uses: peter-evans/create-pull-request@v5
+        with:
+          branch: 'merge/2022.2'
+          commit-message: Merge mps/2022.3 into mps/2022.2
```
Comment (alexanderpann) [inline .github/workflows/PR-into-2022-2.yml:48]: The commit message looks wrong.

**Review by alexanderpann — CHANGES_REQUESTED** [review]
I think there are 2 lines that need to be changed.

File: .github/workflows/PR-into-2022-2.yml (hunk 1 above)
Comment (sergej-koscejev) [reply .github/workflows/PR-into-2022-2.yml:40]: This part is actually only stripping the previous parentheses from the title (to be able to cascade these PRs without the title growing longer). I’ll add a comment to explain this.

**Review by alexanderpann — APPROVED** [review]

**Pull request merged**